tpcx_bb/cluster_configuration/cluster-scheduler.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Are we including this file as an example? Perhaps renaming it `example-scheduler-config.json` or similar would indicate that.

Reply:
Currently, this is used in the cluster setup script. We can change the naming to make it clear this is an example, as it's primarily serving to simply set addresses. @VibhuJawa do we need this?

Reply:
I believe we need a `scheduler.json` file, as `dask-cuda-worker` has/had issues with setting connection information via CLI args

Reply:
I think the fix for this may have come through as a byproduct of the dask-cuda refactor. Either way, let's leave it in for now and switch to your suggested naming convention

Reply:
Resolved by https://github.com/rapidsai/tpcx-bb/pull/11/commits/5f4bd802872a33c475871bf12bb936764555112f